src/zjs_error.c

```diff
@@ -161,7 +161,7 @@ jerry_value_t zjs_custom_error(const char *name, const char *message)
 jerry_value_t zjs_standard_error(zjs_error_type_t type, const char *message)
 {
     int count = sizeof(error_types) / sizeof(zjs_error_t);
-    if (type >= count || type < 0) {
+    if (type >= count) {
```

Comment:
again, clang gives a warning that type is never < 0 since it's unsigned, so I had to drop this.